### DISCUSSION TITLE
월드 버전 업데이트에 관해서 질문드립니다!

### DISCUSSION THREAD

**Discussion by BaePO** [naverz/zepeto-studio-kor#951] [5. Q&A]
안녕하세요!

출시된 월드를 업데이트하려고 하는데 과정에서 질문할 게 있어서 Q&A 올립니다!

패키지 파일을 새로 올리면 기존 월드에 저장된 DB도 같이 유지가 되는 건지 아니면 초기화되는 건지 궁금합니다!

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 서버에 저장하신 Datastorage는 새 버전을 올려도 ZEPETO DB에 저장되어 유지가 됩니다 :) 
클라이언트에 저장하신 데이터는 제페토 앱을 삭제하거나 기기를 변경하는 경우 데이터가 유지되지 않습니다. 
참고 부탁 드립니다.
감사합니다.

**Reply by BaePO** [reply]
감사합니다!

**Reply by BaePO** [reply]
안녕하세요! 이번 대학생 월드 공모전에 참가한 젬토리 팀입니다!

공모전 결과 나온 후에 월드 버전 업데이트하려 했으나 현재 출시된 상태라서 몇몇 버그 수정사항을 업데이트하고 서비스를 하려고 합니다.

혹시 결과가 아직 안나온 지금 업데이트를 하고 심사를 제출해도 다른 불이익이 없을 지 궁금해서 질문드립니다. 감사합니다!

**Reply by Zepeto-TechEvangelist** [reply]
넵, 가능합니다. 불이익은 없습니다.
참고로 공모전 마감 시간에 첫 출시한 팀들이 리젝 사유를 받았을 경우, 수정하시도록 안내 드리고 있습니다.
